src/surfaces/drawer/drawerTitleBar.jsx

```diff
@@ -1,101 +1,178 @@
-import _ from 'lodash';
+import {
+    isObject,
+    isString,
+} from 'lodash';
 import ClassNames from 'classnames';
 import PropTypes from 'prop-types';
-import React from 'react';
-import Header from '../../dataDisplay/header';
+import React, { useEffect, useRef } from 'react';
```

Comment:
**NAB**: This would be my only comment, suggesting a line break when importing there hooks:
```
import React, {
     useEffect,
     useRef,
} from 'react';
```